engine/src/main/java/org/terasology/engine/core/modes/StateLoading.java

```diff
@@ -165,6 +166,7 @@ private void initClient() {
 
     private void initHost() {
         loadProcesses.add(new RegisterMods(context, gameManifest));
+        loadProcesses.add(new InitialiseRendering(context));
```

Comment:
Does this need to be in `initClient` as well?

Reply:
not sure, I just managed to get this to work for single player still need to test things.